app/views/api/metric_visibilities/change.js.erb

```diff
@@ -0,0 +1,19 @@
+<%# If EventTarget is replaceWith'ed immediately, ajax:success and ajax:complete are not called and the spinner will stay %>
+setTimeout(() => {
+  const oldRow = document.getElementById('metric_<%= @metric.id %>');
+  const newRow = '<%= escape_javascript(render(partial: "api/metrics/metric", locals: { metric: @metric })) %>';
+  oldRow.insertAdjacentHTML('afterend', newRow);
+  oldRow.remove();
+});
+
+<% if @usage_limits %>
+  void function () {
+    const slot = document.getElementById('<%= dom_id(@metric) %>_slot');
+
+    if (slot !== null && slot.querySelector('#usage_limits_table')) {
```

Comment:
What would happen if the slot exists, but there is not usage table inside? :upside_down_face: 

Looks like in the original implementation the slot was always re-generated?... :thinking: 

So far I think the slot is always generated correctly.

The weird thing here is that the "slot" is generated as 
`<tr><td class="metric_slot" ...></td></tr>`

So, when it's deleted, only the `td` is removed, but the `tr` stays forever. But I see that it has been like this before, so OK.